### ISSUE TITLE
MQTT disconnected, retrying in 60 seconds!

### ISSUE THREAD

**Issue by iFrez** [nVuln/homebridge-lg-thinq#89]
**Describe The Bug:**
MQTT disconnected from server (not connected)

**Logs:**

![P5FYDmr5SX](https://user-images.githubusercontent.com/2266343/135881837-f47a1e5e-a79c-461b-aec9-c3838fe3293b.png)

**Plugin Config:**

```
        {
            "country": "RU",
            "language": "ru-RU",
            "auth_mode": "token",
            "refresh_token": "",
            "platform": "LGThinQ",
            "devices": [
                {
                    "id": "",
                    "name": "Холодильник",
                    "type": "REFRIGERATOR"
                }
            ],
            "thinq1": true,
            "refresh_interval": 60
        },
```

**Screenshots:**
![47t5fLeSKj](https://user-images.githubusercontent.com/2266343/135880152-85a4543c-7ba6-451e-b52a-a292917556de.png)

**Environment:**

* **Plugin Version**: v1.2.7/v1.2.6/v1.2.5
* **Homebridge Version**: 1.3.4
* **Node.js Version**:  v14.18.0
* **NPM Version**: v6.14.15
* **Operating System**:  Homebridge Raspbian v1.0.14 - Raspberry Pi Zero W Rev 1.1

**Comment by nVuln** [comment]
just checked it, LG from RU is connected to mqtt server `common.iot.ruic.lgthinq.com`, I don't know which aws region of this hostname, normally it's should be `region.iot.amazonaws.com`
I just tried all root certificate I can found on AWS document but nothing work

at this time, you can downgrade to version `v1.1.5` until I found the way to fix this issue, this version not use mqtt protocol

**Comment by nVuln** [comment]
@iFrez I already published test version `v1.2.8-test.2` to fix your issue, can you help me test it ?

**Comment by iFrez** [comment]
Yes, of course. I help you as much as i can) After installing **v1.2.8-test.2** seems all works fine! 
![ChqW8qvC7G](https://user-images.githubusercontent.com/2266343/136029282-34f9d01a-041a-4388-ac2b-ca61fa5e07a6.png)

I am also tested open door sensor and it's work too.
